gradle/verification-metadata.xml

```diff
@@ -755,6 +758,7 @@
       </component>
       <component group="io.github.detekt.sarif4k" name="sarif4k" version="0.0.1">
          <artifact name="sarif4k-0.0.1.jar">
+            <pgp value="8569c95cadc508b09fe90f3002216ed811210daa"/>
```

Comment:
Please remove the line below that says the artifact is not signed ...

Reply:
When I do that I can't get dependency verification to work, for some reason and I can't figure it out... For now, I've removed the PGP key, left only the checksum. Pls. take a look if you can.

Reply:
Fixed.